src/extension/serializer.ts

```diff
@@ -531,7 +545,7 @@ export class GrpcSerializer extends SerializerBase {
     const lid = GrpcSerializer.getDocumentLifecycleId(data.metadata)
     const serialRequest = <SerializeRequest>{ notebook }
 
-    const output = this.cacheNotebookOutputs(notebook, lid)
+    const output = await this.cacheNotebookOutputs(notebook, lid)
```

Comment:
@degrammer why `await` here? this will cause 548 and 549 to no longer run in parallel which is being awaited on 552.